app/api/controllers/requestParsers/validators/validations/JsonFormatValidation.scala

```diff
@@ -48,8 +48,11 @@ object JsonFormatValidation {
       Left(List(RuleIncorrectOrEmptyBodyError))
     } else {
       data.validate[A] match {
-        case JsSuccess(a, _)                                          => Right(a)
-        case JsError(errors: Seq[(JsPath, Seq[JsonValidationError])]) => Left(handleErrors(errors))
+        case JsSuccess(a, _) => Right(a)
```

Comment:
Is there any difference between the one here in the `api` folder and the `JsonFormatValidation` in the `v2` folder? We should probably apply the `sorted` functionality here too, as you've done in the other to keep them the same.